akka-http/src/main/scala/akka/http/server/RejectionHandler.scala

```diff
@@ -130,4 +221,6 @@ object RejectionHandler {
       case (remaining, transformation) ⇒ transformation.transform(remaining)
     }
   }
+
+  private object PrivateRejection extends Rejection
```

Comment:
I use this private rejection for a naive test of whether a given handler has a catchall case.
This test is not perfect, because it doesn't correctly classify things like this:

```scala
handle {
  case x if !x.isInstanceOf[MissingCookieRejection] ⇒ complete(...)
}
```

but it should be good enough for the very large majority of cases.
If I have a really "strange" custom `RejectionHandler` I can still say `myHandler withFallBack default` to seal it manually, so this simple implementation is good enough IMO.
```